<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PNGImageProcessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QGraphicsView" name="graphicsView_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <widget class="QMenu" name="menuNoize">
     <property name="title">
      <string>Noize</string>
     </property>
     <addaction name="actionAWGN"/>
     <addaction name="actionImpulse"/>
    </widget>
    <widget class="QMenu" name="menuContrast">
     <property name="title">
      <string>Contrast</string>
     </property>
     <addaction name="actionGlobalContrastCalibration"/>
    </widget>
    <widget class="QMenu" name="menuFilters">
     <property name="title">
      <string>Filters</string>
     </property>
     <addaction name="action2DConvultion"/>
     <addaction name="actionMedianFilter"/>
     <addaction name="actionNLmeans"/>
    </widget>
    <addaction name="actionInvert"/>
    <addaction name="actionFloyd"/>
    <addaction name="actionResize"/>
    <addaction name="menuNoize"/>
    <addaction name="actionBlend"/>
    <addaction name="menuFilters"/>
    <addaction name="menuContrast"/>
    <addaction name="actionBinarization"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProcessing"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionFloyd">
   <property name="text">
    <string>Floyd</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionAWGN">
   <property name="text">
    <string>AWGN</string>
   </property>
  </action>
  <action name="actionImpulse">
   <property name="text">
    <string>Impulse</string>
   </property>
  </action>
  <action name="actionBlend">
   <property name="text">
    <string>Blend</string>
   </property>
  </action>
  <action name="actionGlobalContrastCalibration">
   <property name="text">
    <string>GlobalContrastCalibration</string>
   </property>
  </action>
  <action name="action2DConvultion">
   <property name="text">
    <string>2DConvultion</string>
   </property>
  </action>
  <action name="actionMedianFilter">
   <property name="text">
    <string>MedianFilter</string>
   </property>
  </action>
  <action name="actionBinarization">
   <property name="text">
    <string>Binarization</string>
   </property>
  </action>
  <action name="actionNLmeans">
   <property name="text">
    <string>NLmeans</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
